remote/gogs/helper.go

```diff
@@ -66,13 +66,29 @@ func buildFromPush(hook *PushHook) *model.Build {
 		hook.Repo.Url,
 		fixMalformedAvatar(hook.Sender.Avatar),
 	)
+
+	eventType := "Undefined"
+	message := "No message"
+
+	switch {
+	case hook.RefType == "tag":
+		eventType = model.EventTag
+		message = "Create Tag " + hook.Ref
+	case hook.RefType == "branch":
+		eventType = model.EventPush
+		message = "Create Branch " + hook.Ref
+	default:
+		eventType = model.EventPush
+		message = hook.Commits[0].Message
+	}
+
 	return &model.Build{
-		Event:     model.EventPush,
+		Event:     eventType,
 		Commit:    hook.After,
 		Ref:       hook.Ref,
 		Link:      hook.Compare,
 		Branch:    strings.TrimPrefix(hook.Ref, "refs/heads/"),
```

Comment:
In gogs the ref sent for a tag creation is just the tag name, so I think here Branch will have the value of the tag. Maybe it would be eaisier to read if Branch was set before in the `switch`